docs/quickstarts/splunk.md

```diff
@@ -0,0 +1,183 @@
+---
+title: Demo Splunk operator with Logging operator
+shorttitle: Splunk HEC
+weight: 300
+---
+
+{{< contents >}}
+
+<p align="center"><img src="../../img/splunk.png" width="340"></p>
+
+This guide describes how to collect application and container logs in Kubernetes using the Logging operator, and how to send them to Splunk.
+
+The following figure gives you an overview about how the system works. The Logging operator collects the logs from the application, selects which logs to forward to the output, and sends the selected log messages to the output (in this case, to Splunk). For more details about the Logging operator, see the [Logging operator overview]({{< relref "docs/one-eye/logging-operator/_index.md">}}).
```

Comment:
The following figure gives you ... : There is no figure here

Reply:
Thanks!